cli/lib/snapshots.ts

```diff
@@ -114,6 +137,7 @@ export async function saveSnapshotToAppdata(
 }
 
 export async function deleteSnapshotFromAppdata( snapshotUrl: string ): Promise< void > {
+	await lock( UPDATE_SNAPSHOTS_LOCKFILE_PATH, { wait: 1000 } );
```

Comment:
I might be missing something here, but aren't we missing some error handling here?
In case of an exception or an early return, doesn't the lock stay in place?